### PR TITLE
Feature/tighter UI

### PR DESCRIPTION
**Pull request by Qblack**
Hi - 
I found this tool and I love it but I plan to use it for something about more microservicesy. 
I found it a bit too spacy when I ended up with  multiple endpoints worth of apps. 
So I made a couple changes which I hope other people will like. 
Ignore the blueprint field - I'm assuming that should be another PR. 

image

### CODE REVIEW

**Comment by alexander34ro** [comment]
@mircealungu I checked the changes and they look good. I see no issues in merging this.

**Review by alexander34ro — APPROVED** [review]

File: flask_monitoringdashboard/frontend/sass/custom.css [hunk 1]
```diff
@@ -507,4 +507,10 @@ footer.sticky-footer {
     color: red;
     vertical-align: super;
     font-size: small;
+}
+
+@media (min-width: 1500px) {
+    .container {
+        max-width: 1600px !important;
```
Comment (mircealungu) [inline flask_monitoringdashboard/frontend/sass/custom.css:514]: shouldn't both these be the same?
or at least the max-width smaller than min-width?

**Comment by mircealungu** [comment]
Did you make the table rows less tall? 
Without a before/after view it's hard for me to imagine the changes :(

**Comment by Qblack** [comment]
> Did you make the table rows less tall?
> Without a before/after view it's hard for me to imagine the changes :(

More compact yeah - I plan to end up with a lot end points in this. Heres a screen shot from the live demo for a before

image

File: flask_monitoringdashboard/frontend/sass/custom.css (hunk 1 above)
Comment (Qblack) [reply flask_monitoringdashboard/frontend/sass/custom.css:514]: Honestly I have no idea. ```@media (min-width: 1200px){.container{max-width:1140px}}``` looks like you are right though. I think I understood it backwards.
I changed the min-width to 1644px and it still looks good for me.

**Pull request merged**